### PR TITLE
Provide pdb file for cydlib/bin on Windows in a `pdb_file` output group

### PR DESCRIPTION
**Pull request by ddeville**
I've chose the `pdb_file` name for the output group since it matches what `cc_binary` uses
(https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/rules/cpp/CcBinary.java;l=707)

There is one edge case that is not covered here: if the user passes `/DEBUG:NONE` in the `rustc_flags` no pdb will be generated and I imagine the build will fail since it's now missing an output. I guess we could check the `rustc_flags` for it but I don't know if it's the right place to do so?

### CODE REVIEW

**Comment by ddeville** [comment]
Looks like the test launcher adds its own `OutputGroupInfo` provider, let me see if those can be combined...

**Comment by UebelAndre** [comment]
The launcher is actually something I think we need to remove. I can ad an incompatibility flag for it

**Comment by UebelAndre** [comment]
> The launcher is actually something I think we need to remove. I can ad an incompatibility flag for it

Sorry this took so long. I've opened https://github.com/bazelbuild/rules_rust/pull/1070

**Review by UebelAndre — APPROVED** [review]
Thanks for putting this together, Awesome work! 😄

File: test/unit/win_interface_library/win_interface_library_analysis_test.bzl [hunk 1]
```diff
@@ -36,6 +36,12 @@ def win_interface_library_analysis_test_suite(name):
         target_compatible_with = ["@platforms//os:windows"],
     )
 
+    native.filegroup(
```
Comment (UebelAndre) [inline test/unit/win_interface_library/win_interface_library_analysis_test.bzl:39]: Can you add a filegroup for a `rust_binary` target as well?

**Review by UebelAndre — APPROVED** [review]
Oh, one last change, if you don't mind 😅

File: test/unit/win_interface_library/win_interface_library_analysis_test.bzl (hunk 1 above)
Comment (ddeville) [reply test/unit/win_interface_library/win_interface_library_analysis_test.bzl:39]: sure can do! now let's see if the tests still pass 🤞🏼

Comment (ddeville) [reply test/unit/win_interface_library/win_interface_library_analysis_test.bzl:39]: ah derp I added it to the wrong binary type, give me a sec.

**Review by UebelAndre — APPROVED** [review]
Thank you so much!

**Pull request merged**